spec/helpers/curate_helper_spec.rb

```diff
@@ -238,25 +238,30 @@ def render_curation_concern_attribute_html(expected_label, *item_or_collection)
   end
 
   describe '#sorted_college_list_for_degrees' do
-    before do
+  before do
       stub_const("COLLEGE_AND_DEPARTMENT",
                   {"current_colleges_for_degrees"=> 
                     {
                       "ceas"=>{"label"=>"Engineering"},
-                      "com"=>{"label"=>"College of Medicine"},
-                      "a&s"=>{"label"=>"Arts & Sciences"}
-                    }
-                  } 
+                      "com"=>{"label"=>"Arts & Sciences"},
```

Comment:
This is a stub so it likely doesn't matter, but I see you removed the A&S pair and changed the 'com' value to A&S, which I think probably doesn't reflect the real data coming in.